evaluation/tutorials/CONUS404/3_CONUS404_DRB_Visualization.ipynb

```diff
@@ -1030,14 +166,14 @@
    "source": [
     "## **Step 2: Visualize and Compare Gridded Data**\n",
     "\n",
-    "Now we're going to create interactive maps of two different gridded datasets. The two different datasets we will be using are the CONUS404 Delaware River Basin subset ('conus404-drb-OSN') and the PRISM Delaware River Basin subset ('prism-drb-OSN'). For the purposes of this notebook, we will only visualize the most recent data in each dataset.\n",
+    "Now we're going to create interactive maps of two different gridded datasets. The two different datasets we will be using are the CONUS404 Delaware River Basin subset (_'conus404-drb-OSN'_) and the PRISM Delaware River Basin subset (_'prism-drb-OSN'_). For the purposes of this notebook, we will only visualize the most recent data in each dataset.\n",
     "\n",
-    "Let's start by loading in our two datasets and combining them into a list."
+    "Let's start by loading in our two datasets and combining them into a list. Note that the timesteps in these datasets are the same (monthly) though have different periods of record, we will use the first time point for comparison."
```

Comment:
```suggestion
    "Let's start by loading in our two datasets and combining them into a list. Note that the timesteps in these datasets are the same (monthly). However, they have different periods of record. We will use the first time point for comparison."
```

Reply:
I was having trouble understanding this sentence and suggested a rewrite to help with clarity.

Reply:
This sounds better, I'll make this change.